l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -107,7 +107,7 @@ def _default_ddt_type(self):
         string='Method of Transportation')
     carrier_id = fields.Many2one(
         'res.partner', string='Carrier')
-    parcels = fields.Integer('Parcels')
+    parcels = fields.Integer('Packages')
```

Comment:
a me personalmente non piace molto la non uniformità tra label e nome del campo. L'ideale sarebbe uno script di migrazione in cui rinominare il campo.

Reply:
Concordo con @tafaRU, però come prima toppa credo si possa fare la semplice modifica della label.

Reply:
Uno script di migrazione sarebbe il top, altrimenti si potrebbe comunque al codice un commento TODO di modifica come "remind" per la prossima versione.

Reply:
Lo terrei così anche per non dover modificare gli  altri moduli che ne fanno uso.

Reply:
> Lo terrei così anche per non dover modificare gli altri moduli che ne fanno uso.

sì corretto, per curiosità: quali sarebbero i moduli che ne fanno uso?

Reply:
Mi pare solo `l10n_it_fatturapa_out_ddt`